.github/workflows/deploy.yml

```diff
@@ -53,11 +66,36 @@ jobs:
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v1
 
-      - name: Build and push the image
+      - name: Set current date as env variable
+        run: echo "TODAY=$(date +'%Y%m%d')" >> $GITHUB_ENV
+
+      - name: Build and push the image (future)
         uses: docker/build-push-action@v3
+        if: github.event_name == 'push' && github.ref == 'refs/heads/main' 
         with:
-          push: ${{ github.ref == 'refs/heads/main' || github.ref == 'refs/heads/future' }}
+          push: true
           platforms: linux/amd64,linux/arm64
           tags: |
-            ${{ secrets.PUBLIC_RUNNER_TERRAFORM_ECR_REPOSITORY_URL }}:${{ github.ref == 'refs/heads/main' && 'latest' || 'future' }}
-            ghcr.io/spacelift-io/runner-terraform:${{ github.ref == 'refs/heads/main' && 'latest' || 'future' }}
+            ${{ secrets.PUBLIC_RUNNER_TERRAFORM_ECR_REPOSITORY_URL }}:future
+            ghcr.io/spacelift-io/runner-terraform:future
+
+      - name: Build and push the image (latest)
+        uses: docker/build-push-action@v3
+        if: github.event_name == 'schedule' || startsWith(github.ref, 'refs/tags/v')
+        with:
+          push: true
+          platforms: linux/amd64,linux/arm64
+          tags: |
+            ${{ secrets.PUBLIC_RUNNER_TERRAFORM_ECR_REPOSITORY_URL }}:latest
+            ${{ secrets.PUBLIC_RUNNER_TERRAFORM_ECR_REPOSITORY_URL }}:${{ steps.latest-tag.outputs.tag }}_${{ env.TODAY }}
+            ghcr.io/spacelift-io/runner-terraform:latest
+            ghcr.io/spacelift-io/runner-terraform:${{ steps.latest-tag.outputs.tag }}_${{ env.TODAY }}
+
+      - name: Create release (latest)
+        uses: ncipollo/release-action@v1
+        if: github.event_name == 'schedule' || startsWith(github.ref, 'refs/tags/v')
```

Comment:
Do we really want to create a release when the cronjob triggers? Won't it end up with empty release notes? I guess if we don't it's maybe confusing because we'll have published a new version without a release.

Reply:
Yeah it’ll be an empty note, but my thought was your last sentence basically.

> if we don't it's maybe confusing because we'll have published a new version without a release

Reply:
Can we set the note to "Weekly rebuild" or something like that?

Reply:
👍